packages/@webex/plugin-meetings/test/unit/spec/meetings/index.js

```diff
@@ -1358,6 +1361,28 @@ describe('plugin-meetings', () => {
             );
           });
 
+          it('creates the meeting avoiding meeting info fetch by passing one2OneMeeting true', async () => {
+
+            const meeting = await webex.meetings.createMeeting('test destination', 'test type', false, {}, undefined, false, undefined, undefined, true);
+
+            const expectedMeetingData = {
+              permissionToken:
+                'eyJhbGciOiJIUzI1NiJ9.eyJleHAiOiIxMjM0NTYiLCJwZXJtaXNzaW9uIjp7InVzZXJQb2xpY2llcyI6eyJhIjp0cnVlfX19.wkTk0Hp8sUlq2wi2nP4-Ym4Xb7aEUHzyXA1kzk6f0V0',
```

Comment:
Let us please refrain from adding any kind of tokens to UTs as it is a publicly maintained code. Please change this to `FAKE_TOKEN` and then update tests accordingly